aptly/publisher/__init__.py

```diff
@@ -186,6 +186,91 @@ def _get_publish(self):
                 return publish
         raise NoSuchPublish("Publish %s does not exist" % self.name)
 
+    def save_publish(self, aptly_dir="/srv/aptly/"):
```

Comment:
Don't use any default for output directory, that should be mandatory option, or default to current working directory